src/OrdinaryDiffEq.jl

```diff
@@ -257,6 +257,9 @@ export OrdinaryDiffEqExponentialRK, LawsonEuler, NorsettEuler, ETD1, ETDRK2, ETD
        EPIRK4s3B,
        EPIRK5s3, EXPRB53s3, EPIRK5P1, EPIRK5P2, ETD2, Exprb32, Exprb43
 
+using OrdinaryDiffEqNewmark
+export NewmarkBeta
+
```

Comment:
we don't want this to be an OrdinaryDiffEq dep. We're about to remove most of the other sub-libraries.